lib/rollout/ui/web.rb

```diff
@@ -45,7 +45,20 @@ class Web < Sinatra::Base
           if params[:users]
             feature.users = params[:users].split(',').map(&:strip).uniq.sort
           end
-          feature.data.update(description: params[:description])
+          feature.data.each do |name, old_val|
+            new_val = params[name]
+
+            # keep type the same (for integers/boolean/strings)
+            new_val = if old_val.is_a? Integer
```

Comment:
I think it would be more bulletproof and ellegant to use a regexp to see if it's a number, eg https://medium.com/launch-school/number-validation-with-regex-ruby-393954e46797